test/event_test.py

```diff
@@ -28,6 +28,70 @@
 )
 
 
+NAMES_AND_EVENTS = (
+    ('NoEvent', pygame.NOEVENT),
+    ('ActiveEvent', pygame.ACTIVEEVENT),
+    ('KeyDown', pygame.KEYDOWN),
+    ('KeyUp', pygame.KEYUP),
+    ('MouseMotion', pygame.MOUSEMOTION),
+    ('MouseButtonDown', pygame.MOUSEBUTTONDOWN),
+    ('MouseButtonUp', pygame.MOUSEBUTTONUP),
+    ('JoyAxisMotion', pygame.JOYAXISMOTION),
+    ('JoyBallMotion', pygame.JOYBALLMOTION),
+    ('JoyHatMotion', pygame.JOYHATMOTION),
+    ('JoyButtonDown', pygame.JOYBUTTONDOWN),
+    ('JoyButtonUp', pygame.JOYBUTTONUP),
+    ('VideoResize', pygame.VIDEORESIZE),
+    ('VideoExpose', pygame.VIDEOEXPOSE),
+    ('Quit', pygame.QUIT),
+    ('SysWMEvent', pygame.SYSWMEVENT),
+
+    ('UserEvent', pygame.USEREVENT),
+    ('UserEvent', pygame.USEREVENT + 1),
+    ('UserEvent', pygame.NUMEVENTS - 1),
+
+    ('Unknown', 0xFFFF),
+)
+
+# Add in any SDL 2 specific events.
+if pygame.get_sdl_version()[0] >= 2:
+    NAMES_AND_EVENTS += (
+        ('FingerMotion', pygame.FINGERMOTION),
+        ('FingerDown', pygame.FINGERDOWN),
+        ('FingerUp', pygame.FINGERUP),
+        ('MultiGesture', pygame.MULTIGESTURE),
+
+        # These can be corrected when issue #1221 is resolved.
+        # Should be: 'AudioDeviceAdded'
+        ('Unknown', pygame.AUDIODEVICEADDED),
```

Comment:
Seems these are brought in with 2.0.4. 
https://wiki.libsdl.org/SDL_AudioDeviceEvent
`This structure is available since SDL 2.0.4.`

Reply:
I've made the requested change. Thanks.